marlowe/src/Language/Marlowe/Pretty.hs

```diff
@@ -78,8 +78,8 @@ instance (Pretty1 a, Pretty1 b) => Pretty1 (a :*: b) where
     pretty1 topLevel (f :*: g) = pretty1 topLevel f <> pretty1 topLevel g
     isNullary _ = False
 
-instance Pretty String where
-  prettyFragment = text
+instance {-# OVERLAPPING #-} Pretty [Char] where
+  prettyFragment = text . show
```

Comment:
what's this?

Reply:
Because there is already an instance of Pretty [a], there are overlapping instances apparently. And this is what I was able to do to get it to work without errors/warnings. Also I added show cause before pretty printing wasn't adding the quotes around the string

Reply:
how come it used to work though?

Reply:
I think it is because we didn't use the `Pretty String` instance. We had `Text` everywhere (for `Role` ids and such)